cg/meta/transfer/external_data.py

```diff
@@ -144,43 +144,51 @@ def add_files_to_bundles(
             LOG.info("Adding path %s to bundle %s in housekeeper" % (path, lims_sample_id))
             self.housekeeper_api.add_file(path=path, version_obj=last_version, tags=HK_FASTQ_TAGS)
 
+    def get_failed_samples(self, fastq_paths_to_add: List[Path]) -> List[Path]:
```

Comment:
I think this should be renamed since you are not getting the failed samples but the failed fastq paths for a sample